.gitignore

```diff
@@ -29,3 +29,4 @@ web.*.config
 waca.sln.DotSettings.user
 .vs/*
 .arc*
+password.php
```

Comment:
I think @stwalkerster is saying that you shouldn't add this file to .gitignore since you're the only person who needs it.  Instead, add it to `.git/info/exclude`, which is like .gitignore but only for you and never pushed to the rest of the repository.

Also, on the subject of whitespace, please remember to include a blank line at the end of files (with the exception of minified JS and CSS files).